### DISCUSSION TITLE
When is `search.suggest` feature going to be available to the public please ?

### DISCUSSION THREAD

**Discussion by akharrou** [squidfunk/mkdocs-material#2302] [Q&A] [answered]
I checked on the Mkdocs Material website and it said that [`search.suggest`](https://squidfunk.github.io/mkdocs-material-insiders/setup/setting-up-site-search/#search-suggestions) hit the funding goal and [is checked as green](https://squidfunk.github.io/mkdocs-material-insiders/insiders/#2500-biquinho-vermelho). When can we expect it to be released to the public (because it still says only for Insiders) ? Thanks

**Comment by martinbira** [comment, marked as answer]
I assume when the funding goal is complete as stated on https://squidfunk.github.io/mkdocs-material/insiders/#funding

**Reply by squidfunk** [reply]
That's correct. The green check mark means that the feature is implemented and ready for use. Compare for example the "code palette toggle" which is planned, but not implemented yet. Thus, search suggestions are tied to the $ 2,500 funding goal, which might be reached later this year 😊
